src/SqlAsyncCollector.cs

```diff
@@ -542,7 +544,8 @@ AS NewData
             /// <returns>TableInformation object containing primary keys, column types, etc.</returns>
             public static async Task<TableInformation> RetrieveTableInformationAsync(SqlConnection sqlConnection, string fullName, ILogger logger, IEnumerable<string> objectColumnNames)
             {
-                Dictionary<TelemetryPropertyName, string> sqlConnProps = sqlConnection.AsConnectionProps();
+                // pass null for ServerProperties since it's already logged before this call.
```

Comment:
I would still pass the connection params so that anywhere we add any connection params we add as many as we can - no reason to leave some out some of the time.

Reply:
connection params? That's read from the connection object itself, I don't need to pass it explicitly, am I missing something here?

Reply:
Whoops, meant server properties 😄